hamilton/io/fingerprinting.py

```diff
@@ -0,0 +1,178 @@
+import base64
+import functools
+import hashlib
+import logging
+from collections.abc import Mapping, Sequence, Set
+from typing import Dict
+
+from hamilton.experimental import h_databackends
+
+logger = logging.getLogger(__name__)
+
+
+MAX_DEPTH = 2
```

Comment:
So the pattern is for people to change it? Might be worth having a method to do that. Or, add a TODO to make it easier to change. Generally bad to have people alter a constant in a module for config.